core/docs/config.inc.tpl

```diff
@@ -1,87 +1,87 @@
-    define('MODX_LOG_LEVEL_WARN', 2);
-    define('MODX_LOG_LEVEL_INFO', 3);
-    define('MODX_LOG_LEVEL_DEBUG', 4);
-}
+<?php
+/**
+ *  MODX Configuration file
+ */
+$database_type = '{database_type}';
+$database_server = '{database_server}';
+$database_connection_charset = '{database_connection_charset}';
+$table_prefix = '{table_prefix}';
+$dbase = '{dbase}';
+$database_user = '{database_user}';
+$database_password = '{database_password}';
+$database_dsn = '{database_dsn}';
+$config_options = {config_options};
+$driver_options = {driver_options};
+
+$lastInstallTime = {last_install_time};
+
+$site_id = '{site_id}';
+$site_sessionname = '{site_sessionname}';
+$https_port = '{https_port}';
+$uuid = '{uuid}';
+
+if (!defined('MODX_CORE_PATH')) {
+    $modx_core_path= '{core_path}';
+    define('MODX_CORE_PATH', $modx_core_path);
+}
+if (!defined('MODX_PROCESSORS_PATH')) {
+    $modx_processors_path= '{processors_path}';
+    define('MODX_PROCESSORS_PATH', $modx_processors_path);
+}
+if (!defined('MODX_CONNECTORS_PATH')) {
+    $modx_connectors_path= '{connectors_path}';
+    $modx_connectors_url= '{connectors_url}';
+    define('MODX_CONNECTORS_PATH', $modx_connectors_path);
+    define('MODX_CONNECTORS_URL', $modx_connectors_url);
+}
+if (!defined('MODX_MANAGER_PATH')) {
+    $modx_manager_path= '{mgr_path}';
+    $modx_manager_url= '{mgr_url}';
+    define('MODX_MANAGER_PATH', $modx_manager_path);
+    define('MODX_MANAGER_URL', $modx_manager_url);
+}
+if (!defined('MODX_BASE_PATH')) {
+    $modx_base_path= '{web_path}';
+    $modx_base_url= '{web_url}';
+    define('MODX_BASE_PATH', $modx_base_path);
+    define('MODX_BASE_URL', $modx_base_url);
+}
+if(defined('PHP_SAPI') && (PHP_SAPI == "cli" || PHP_SAPI == "embed")) {
+    $isSecureRequest = false;
+} else {
+    $isSecureRequest = ((isset($_SERVER['HTTPS']) && !empty($_SERVER['HTTPS']) && strtolower($_SERVER['HTTPS']) !== 'off') || $_SERVER['SERVER_PORT'] == $https_port);
+}
+if (!defined('MODX_URL_SCHEME')) {
+    $url_scheme=  $isSecureRequest ? 'https://' : 'http://';
+    define('MODX_URL_SCHEME', $url_scheme);
+}
+if (!defined('MODX_HTTP_HOST')) {
+    if(defined('PHP_SAPI') && (PHP_SAPI == "cli" || PHP_SAPI == "embed")) {
+        $http_host='{http_host}';
+        define('MODX_HTTP_HOST', $http_host);
+    } else {
+        $http_host= array_key_exists('HTTP_HOST', $_SERVER) ? htmlspecialchars($_SERVER['HTTP_HOST'], ENT_QUOTES) : '{http_host}';
+        if ($_SERVER['SERVER_PORT'] != 80) {
+            $http_host= str_replace(':' . $_SERVER['SERVER_PORT'], '', $http_host); // remove port from HTTP_HOST
+        }
+        $http_host .= ($_SERVER['SERVER_PORT'] == 80 || $isSecureRequest) ? '' : ':' . $_SERVER['SERVER_PORT'];
+        define('MODX_HTTP_HOST', $http_host);
+    }
+}
+if (!defined('MODX_SITE_URL')) {
+    $site_url= $url_scheme . $http_host . MODX_BASE_URL;
+    define('MODX_SITE_URL', $site_url);
+}
+if (!defined('MODX_ASSETS_PATH')) {
```

Comment:
@opengeek I apologize for not paying attention to this.
Maybe the `MODX_ASSETS_PATH` variable should be placed higher, before other path variables, since we are correcting the order? Because other variables with `_PATH` are located above.
Could you sort or group them somehow more competently? You know better the priority of variables and their logic.
p.s. I understand that the PR is not really about this, but it would add convenience and logic...